bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -456,13 +456,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
     
     Frame label="Alarm"
     {
-         Switch  item=iAlarm_System_State       label="Alarm state"
```

Comment:
Why did you remove this element ?

Reply:
I am not able to get of those unwanted spaces! any idea?

Reply:
can you just copy+paste the line from a previus version/commit so that you restore the line as it was before?

Reply:
Ok now only the line number 460 looks changed